src/AutoMapper/AdvancedConfiguration.cs

```diff
@@ -40,5 +40,11 @@ public class AdvancedConfiguration
         public List<IExpressionResultConverter> QueryableResultConverters { get; } = ExpressionBuilder.DefaultResultConverters();
 
         public List<IExpressionBinder> QueryableBinders { get; } = ExpressionBuilder.DefaultBinders();
+
+        /// <summary>
+        /// How many levels deep should recursive queries be expanded.
+        /// Must be zero for EF6. Can be greater than zero for EF Core.
+        /// </summary>
+        public int RecursiveQueriesMaxDepth { get; set; }
```

Comment:
A little late perhaps, but maybe we should move the `ProjectTo` configuration to an inner object? I think this stuff doesn't even warrant a major version bump. Just sayin' :)

Reply:
Ah yeah, a little late...